15/umbraco-cms/fundamentals/data/defining-content/document-type-localization.md

```diff
@@ -75,21 +72,25 @@ The localizations are applied by using the syntax `#{area alias}_{key alias}`.
 * Create a new **tab** called `#tabs_content`.
 * Add a new **group** called `#groups_titles`.
 * Add a **property** called `#properties_title` with **alias** `title`.
-  * Set description to `#properties_title-desc`.
+  * Set description to `{#properties_title-desc}`.
   * Use a `TextString` editor.
   * Enable to `Set this field as mandatory`.
   * Under validation add `#properties_title-message`.
 
+{% hint style="info" %}
+Please note, property descriptions support [Umbraco Flavored Markdown](../../../reference/umbraco-flavored-markdown). This requires a different syntax (wrapped in brackets), so not to conflict with Markdown header syntax.
```

Comment:
```suggestion
Property descriptions support [Umbraco Flavored Markdown](../../../reference/umbraco-flavored-markdown), which uses a different syntax (wrapped in brackets) to avoid conflicts with Markdown headers.
```